mitodl/micromasters
# Added order reference numbers

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #5213 

#### What's this PR do?

This adds the `reference_number` column and the code to populate that value on create going forward.

#### How should this be manually tested?

- Go to your dashboard and perform a checkout
- You only need to get as far as being redirected over to CyberSource (you don't need a functional Cybersource integration), once that happens you can go to django-admin and confirm your order has a reference number populated
